docs/development.md

```diff
@@ -203,8 +239,9 @@ When new git tag is created, Travis-ci deploy job automatically builds binaries
 4. Verify that packages have been uploaded to rpm and deb repositories.
 5. Confirm the binaries are available in GitHub release page and update the file `build/VERSION` with latest version number.
 
-## odo-bot
-This is GitHub user that does all the automation.
+## Odo Bot
+
+[odo-bot](https://github.com/odo-bot) is the GitHub user that provides automation for certain tasks of Odo.
```

Comment:
Can we detail out what tasks?
At least a few examples?

Reply:
It is already there. Check next section "Scripts using odo-bot"

Reply:
@anmolbabu do you think it is enough? Or there should be more?
Feel free to merge this is you are OK with it.